src/ModelTable/ModelTableRow.tsx

```diff
@@ -38,7 +39,11 @@ export const ModelTableRow = ({
   return (
     <FormStoreProvider id={data.id} {...formMethods}>
       <Lenses initialLens={DataLens.DISPLAY}>
-        <TableRow prefilled={false} {...props}>
+        <TableRow
+          prefilled={false}
+          className={`${data.id === idToFocus && 'bg-blue-200'}`}
```

Comment:
Choose a color variable or create a new color var.

Reply:
Ah there's even an `accent` color already set, changed! Thanks for the catch.